shared/constants/swaps.ts

```diff
@@ -127,11 +132,12 @@ export const WMATIC_CONTRACT_ADDRESS =
   '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270';
 export const WAVAX_CONTRACT_ADDRESS =
   '0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7';
-
 export const WETH_OPTIMISM_CONTRACT_ADDRESS =
   '0x4200000000000000000000000000000000000006';
 export const WETH_ARBITRUM_CONTRACT_ADDRESS =
   '0x82af49447d8a07e3bd95bd0d56f35241523fbab1';
+export const WETH_ZKSYNC_CONTRACT_ADDRESS =
+  '0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91';
```

Comment:
Just checking, is this supposed to be a checksummed address or lower case?

Reply:
It can be both, since it's converted to lowerCase e.g. here:
https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/swaps/swaps.util.ts#L606

Or here: https://github.com/MetaMask/metamask-extension/blob/develop/shared/lib/swaps-utils.js#L189